tests/clickhouse-test

```diff
@@ -292,6 +292,7 @@ def clickhouse_execute_http(
         "http_receive_timeout": timeout,
         "http_send_timeout": timeout,
         "output_format_parallel_formatting": 0,
+        "max_rows_to_read": 0,  # Some queries read from system.text_log which might get too big
```

Comment:
Ok. Although I don't understand this change - if we unlock it in some tests, what is the point for unlocking it globally here?

PS. Anyways, the changes LGTM. Thank you!

Reply:
AFAIK this is not global, it's only used when executing SQL queries inside clickhouse-test (for internal usage).